docs/_docs/best-practices/compute/compute.md

```diff
@@ -18,48 +18,36 @@ Discover essential FinOps best practices to optimize cost efficiency and governa
 <details open markdown="1">
    <summary class="fs-2 text-uppercase">On this page</summary>
 
-- [Azure Kubernetes Service](#azure-kubernetes-service)
+- [Advisor](#advisor)
 - [Virtual machines](#virtual-machines)
+- [App Service plans](#app-service-plans)
+- [Azure Kubernetes Service](#azure-kubernetes-service)
 - [🙋‍♀️ Looking for more?](#️-looking-for-more)
 - [🧰 Related tools](#-related-tools)
 
 </details>
 
 ---
 
-## Azure Kubernetes Service
+## Advisor
 
-### Query: AKS Cluster
+### List of cost recommendations for Compute
 
-This Azure Resource Graph (ARG) query retrieves detailed information about Azure Kubernetes Service (AKS) clusters within your Azure environment.
+This Azure Resource Graph (ARG) query retrieves a list of Azure Advisor recommendations specifically for compute resources. It filters the recommendations to include only those related to virtual machines, scale sets, and other compute services, providing insights into potential cost savings.
 
 <h4>Category</h4>
 
-Resource management
+Cost optimization
 
 <h4>Query</h4>
 
 ```kql
```

Comment:
nit: We should establish some KQL language standards. If you happen to know of any, we should document them in the wiki. No worries if you haven't.